installer/windows/invest_installer.nsi

```diff
@@ -406,7 +406,7 @@ SectionEnd
 Section "uninstall"
   ; Need to enforce execution level as admin.  See
   ; nsis.sourceforge.net/Shortcuts_removal_fails_on_Windows_Vista
-  SetShellVarContext all
+  SetShellVarContext current

```

Comment:
Can you confirm that the uninstall worked? On Vista? ;) Kidding, can we remove that comment?

Reply:
Hahah, yes WE CAN. This DOES bring up an interesting point... This might only work on Windows 10, not 7... Are we supporting Windows 7 still? It looks like Microsoft doesn't anymore, so I'm inclined not to.
https://support.microsoft.com/en-us/windows/windows-7-support-ended-on-january-14-2020-b75d4580-2cc7-895a-2c9c-1466d9a53962

Reply:
Oh, and yes, I was able to uninstall the program. It'd be great if someone else who has access to a non-admin user could try as well I suppose.

Reply:
Removed old comments and added a new one: https://github.com/natcap/invest/pull/484/commits/d1db4d6fc9cbad86d02ada28e9594f8307a2db79

Reply:
Okay I'll give it a try as well.

Reply:
Well I made a non-admin account and installed the artifact from this PR. I tried the uninstall shortcut that's in the installation directory and Windows kept telling me that it can't uninstall because InVEST is still running. But I also got that when I tried the same from my Admin account with an old version of invest. 

I've probably never tried that shortcut before. I usually go to "Add/Remove Programs" menu on Windows. Checking that menu on either account - it does not even list the newly installed invest (from this PR's build). The installation seems to work though - I could open a model.

Reply:
>I tried the uninstall shortcut that's in the installation directory and Windows kept telling me that it can't uninstall because InVEST is still running

Huh, I didn't have any problem uninstalling this way.

>I usually go to "Add/Remove Programs" menu on Windows

I don't think I've tried this for InVEST! I'll see if I get the same behavior.

Reply:
Ugh, I can't seem to get the installer to download from github actions, so I can't test this out right now (I'll try again later).

Hm, this is interesting: the NSIS docs for [`RequestExecutionLevel`](https://nsis.sourceforge.io/Reference/RequestExecutionLevel) suggest that if we need to write to HKLM in the windows registry (the InVEST installer does), then the `admin` level might be required.  So @davemfish, maybe that's why you aren't able to see it in the Add/Remove programs list?  That list is populate from the Windows Registry.

In any case, it sounds like even if the user doesn't have admin privileges, then the only difference is that they won't be able to uninstall InVEST from add/remove programs.

> Are we supporting Windows 7 still? It looks like Microsoft doesn't anymore, so I'm inclined not to.

Yeah, I don't think we should deliberately support Windows 7 given its age.  It might well still work, but no need to go out of our way to verify it does.

Reply:
I also don't see InVEST in the `Add/Remove Programs` tool for non-admin. Let me see if I can fix that.

Reply:
I wouldn't be surprised if I just happened to get Windows into a state where it thought invest was still running - that's wayy too common on Windows - as opposed to some property of the new installer actually being responsible.

Reply:
In truth, there are lots of reasons why that could be!  The uninstaller only ever looks for a process named `invest.exe`, which could be from a totally different version, could be a zombie process, or could even be something not related to InVEST at all!  It's kind of a crude check.  I wouldn't be opposed to removing it.